src/components/illustration/illustration.tsx

```diff
@@ -48,7 +48,7 @@ export const Illustration = (props: IllustrationProps) => {
   const IllustrationSvg = IllustrationSet[props.show];
 
   return (
-    <IllustrationLayout size={props.size ?? IllustrationSize.NORMAL} {...props}>
```

Comment:
leftover from tinkering round with my component it the lead component context where I tried to style our Illustration component with styled components which forwards a `className`. We still might want to support styled components by adding a className to our components but that's a separate discussion.